docs/rules/require-hook.md

```diff
@@ -148,3 +148,74 @@ afterEach(() => {
   clearCityDatabase();
 });
 ```
+
+## Options
+
+Some test utils provides methods which takes hook as an argument 
+and should be executed outside a hook.
+
+For example https://vue-test-utils.vuejs.org/api/#enableautodestroy-hook 
+which takes the hook as an argument. To exclude them you can update settings
```

Comment:
The fact that the methods take hooks as an argument is irrelevant, so this should be say something along the lines of "if there are methods that you want to call outside of hooks and tests, you can mark them as allowed using the `allowedFunctionCalls` option" :)

Reply:
I updated the documentation but I still don't like it. Because the misleading option `allowedFunctionCalls`, which contain correct function, but the code example is still incorrect because of another function calls. Is it good?

Reply:
> Because the misleading option allowedFunctionCalls,

I'm not sure I follow you on what you're finding misleading?

I'm happy if you want to play with the documentation some more - do you think it would be better if we didn't have _any_ config in our "incorrect" example (e.g. "Example of incorrect code without using `allowedFunctionCalls`"), and left the "correct" example as-is?

Reply:
@G-Rath yes, the problem is just with "incorrect" example. So in docs we are saying "incorrect usage of allowed functions" but there is no incorrect use of these functions in the example. This makes some confusion so I'll remove it